x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/phases/phase/phase.tsx

```diff
@@ -43,8 +44,18 @@ interface Props {
   topLevelSettings?: React.ReactNode;
 }
 
+/**
+ * To re-orient the user about their place in the form after returning from somewhere else,
+ * like the rollup wizard, we want to scroll them back to a specific field. Fields are often hidden
+ * inside of an accordion which has an animation when it opens. We need to wait for this animation
+ * to complete so that the field is in it's final position before we can start scrolling to it.
+ */
+const EUI_ACCORDION_EXPAND_DELAY = 500;
```

Comment:
Would it be possible to export this value from EUI directly? In case it changes or any other settings affect the value.

Reply:
This is actually not a value from the lib directly, it is tied to this value though: https://github.com/elastic/eui/blob/fe426f6f6828b5e9ed88dc82627c54e403801b59/src/global_styling/variables/_animations.scss#L8.

In future this could change, which makes this a slight hack, not sure how best to manage getting around the animation without this though :(